packages/@manta-style/core/src/index.ts

```diff
@@ -1,10 +1,20 @@
 import * as annotationUtils from './utils/annotation';
 import { PluginSystem } from './plugin';
+import { Annotation } from './utils/annotation';
 
-export { Annotation } from './utils/annotation';
 export type MantaStyleContext = {
   query: { [key: string]: unknown };
   plugins: PluginSystem;
 };
 export * from './plugin';
-export { annotationUtils };
+export { annotationUtils, Annotation };
+
+export abstract class Type {
+  abstract deriveLiteral(
```

Comment:
move type definition into `type` folder? 
so the `index.ts` is just import and export?

Reply:
eventually we will do so. due to TypeScript bug, we are unable to have a file containing only types.

Reply:
oops. okay then

Reply:
it's actually a bug in `awesome-typescript-loader`. maybe we could arrange some time to do this refactor.